srcpkgs/peaclock/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'peaclock'
+pkgname=peaclock
+version=0.1.3
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=debug"
```

Comment:
Why do you set the build type to debug? Release is normally what we want and is also the default of the cmake build_style, so remove this line.

Reply:
I thought that binaries should include debug info?

Reply:
I checked when I added it, and Release mode did strip the binaries.

Reply:
Ah, I forgot about that build type. I've removed it now.

Reply:
#7004